src/common/json/json_writer.go

```diff
@@ -0,0 +1,345 @@
+package json
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/bridgecrewio/yor/src/common/types"
+
+	"github.com/bridgecrewio/yor/src/common/logger"
+	"github.com/bridgecrewio/yor/src/common/structure"
+	"github.com/bridgecrewio/yor/src/common/utils"
+)
+
+// readFilePath: origin file path
+// blocks: resources blocks, some containing updated tags
+// writeFilePath: destination for writing the updated json
+// fileBracketsPairs: mapping of all brackets in file by their start char index
+// The function updates the content of `readFilePath` with updated tags from `blocks` and writes it to `writeFilePath`
```

Comment:
@rotemavni I believe godoc comments should start with the function name, i.e. `WriteJSONFile`